coops-ui/tests/unit/AnnualReport.spec.ts

```diff
@@ -1175,10 +1199,6 @@ describe('AnnualReport - Part 6 - Error/Warning dialogues', () => {
     window.location = { assign: jest.fn() } as any
   })
 
-  afterAll(() => {
```

Comment:
Why is this code deleted? Its purpose is to restore the subject function.

From experience, many times I found that the tests:
- run concurrently
- are not independent from each other
- are not independently scoped

Thus, cleaning up after a test's execution is important.

Reply:
Indeed they do, it wasn't required but isn't destructive. Either way, added it back.

Reply:
Code restored, thanks.